src/test/validation_tests.cpp

```diff
@@ -120,20 +120,4 @@ BOOST_AUTO_TEST_CASE(signet_parse_tests)
     BOOST_CHECK(!CheckSignetBlockSolution(block, signet_params->GetConsensus()));
 }
 
-static bool ReturnFalse() { return false; }
-static bool ReturnTrue() { return true; }
-
-BOOST_AUTO_TEST_CASE(test_combiner_all)
-{
-    boost::signals2::signal<bool (), CombinerAll> Test;
```

Comment:
Is the `#include <boost/signals2/signal.hpp>` still needed, now that this is removed?

Reply:
indeed not. Good spot. Now removed